pyproject.toml

```diff
@@ -109,7 +110,7 @@ addopts = """
 -v
 --durations=0
 --show-capture=all
--n auto
+-n 4
```

Comment:
Github runners use 2 or 4. With `auto`, I get license failure locally due to the spawning of too many Fluent processes.